.puppet-lint.rc

```diff
@@ -2,5 +2,6 @@
 --relative
 --no-class_inherits_from_params_class-check
 --no-80chars-check
```

Comment:
According to Nick Miller, if we have no-140chars-check, we don't need to specify no-80chars-check